src/ToricVarieties/LineBundles/auxilliary.jl

```diff
@@ -0,0 +1,24 @@
+
+function command_string(v::AbstractNormalToricVariety, c::Vector{fmpz})
+    # Initialize a list of strings, which we will eventually join to form the command string
+    string_list = Vector{String}()
+    
+    # cohomCalg cannot handle "[" nor "]" in coordinate names, so we remove those characters.
+    # Note that "[" and "]" appear at other places of the command string.
+    
+    # Add information about Cox ring and its grading to string_list
+    vars = [replace(replace(string(x), "[" => ""), "]" => "") for x in Hecke.gens(cox_ring(v))]
```

Comment:
You are right. There is trouble in store (whenever variable names are being changed). So I have indeed just added a check for possible duplicates. The best that I could think of right now was to compare the lengths of `vars` and `unique(vars)` (which I believe removes the duplicates). If there is a better option, please let me know.

Reply:
The way I'd go about this is to store a dictionary which maps "our" variable names to safe variable names for the external tool (i.e., satisfying whatever rules that tool imposes). For debugging it can of course be useful to map names to similar names, but then my rule would be something like this (assuming this is compatible with the rules of the external tool, which I don't know): if the name is fully alphanumeric, map it to itself (then there is no need to store it in the dictionary); otherwise, map it to a sequential lists of variable names, e.g. `_x1, _x2, ...` (this assumes that a leading underscore is permissible).

Of course you can implement more complex transformation rules, but what's the point?

Reply:
That is a good suggestion. I have just added such a dictionary.